histomicstk/Membraneness.py

```diff
@@ -4,10 +4,10 @@
 from Hessian import Hessian
```

Comment:
@slee172 @cooperlab There are already a bunch of functions for this in scikit-image.
* [Hessian matrix](http://scikit-image.org/docs/dev/api/skimage.feature.html#skimage.feature.hessian_matrix)
* [Eigen values of the Hessian matrix](http://scikit-image.org/docs/dev/api/skimage.feature.html#skimage.feature.hessian_matrix_eigvals)
* [determinant of the hessian matrix](http://scikit-image.org/docs/dev/api/skimage.feature.html#skimage.feature.hessian_matrix_det)

Also numpy already has a way to compute [eigen values and vectors](http://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.eig.html#numpy.linalg.eig) for [multiple matrices at once](http://docs.scipy.org/doc/numpy/reference/routines.linalg.html#linear-algebra-on-several-matrices-at-once).